pkg/prow/config/config.go

```diff
@@ -139,14 +139,14 @@ type Controller struct {
 	// JobURLTemplateString compiles into JobURLTemplate at load time.
 	JobURLTemplateString string `json:"job_url_template,omitempty"`
```

Comment:
It seems confusing, there is still `jobs` in the names.

Reply:
this PR doesn't touch the prow config - I figured we'd tackle cleaning that up in a separate PR